pkg/aggregator/aggregator.go

```diff
@@ -122,6 +154,14 @@ var (
 		nil, "Count of hostname update")
 	tlmDogstatsdContexts = telemetry.NewGauge("aggregator", "dogstatsd_contexts",
 		nil, "Count the number of dogstatsd contexts in the aggregator")
+	tlmHugeSeries = telemetry.NewCounter("aggregator", "huge_series",
+		nil, "Count of timeseries with over 100 tags")
+	tlmHugeDistributions = telemetry.NewCounter("aggregator", "huge_distributions",
+		nil, "Count of distributions with over 100 tags")
+	tlmAlmostHugeSeries = telemetry.NewCounter("aggregator", "almost_huge_series",
+		nil, "Count of timeseries with over 90 tags")
+	tlmAlmostHugeDistributions = telemetry.NewCounter("aggregator", "almost_huge_distributions",
```

Comment:
Let's name this "sketch" instead of "distribution", to be consistent with all the other telemetry we expose for sketches/distributions.

Let's add the expvars as well.